persist/sqlite/v2consensus_test.go

```diff
@@ -3,15 +3,35 @@ package sqlite_test
 import (
 	"testing"
 	"time"
+	"unsafe"
 
 	"go.sia.tech/core/consensus"
 	"go.sia.tech/core/types"
 	"go.sia.tech/coreutils/chain"
 	"go.sia.tech/explored/explorer"
 	"go.sia.tech/explored/internal/testutil"
+	"golang.org/x/crypto/blake2b"
 )
 
+// from RFC 6962
+const leafHashPrefix = 0
+const nodeHashPrefix = 1
+
+func hashBlockGeneric(msg *[64]byte, prefix uint64) [32]byte {
```

Comment:
I'd prefer it if we could avoid copying this code over from core and reuse the exposed rhp2 code.

e.g.  the following should work for your test

```go
	data := make([]byte, 2*rhp2.LeafSize)
	data[0], data[rhp2.LeafSize] = 1, 1
	v1FC.FileMerkleRoot, _ = rhp2.ReaderRoot(bytes.NewReader(data))
```

Always better to use the code we also use in production where possible.